app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -35,40 +37,40 @@ class MainActivity: BaseActivity() {
 
     private val mOnNavigationItemSelectedListener =
             BottomNavigationView.OnNavigationItemSelectedListener { item ->
-        when (item.itemId) {
```

Comment:
I'm not sure why is this entire block indented forward. Please make sure to reformat your code using Android Studio before committing changes.